src/NServiceBus.AcceptanceTests/Routing/SubscriptionBehavior.cs

```diff
@@ -38,10 +38,15 @@ public SubscriptionBehavior(Action<SubscriptionEventArgs, TContext> action, TCon
             var subscriptionMessageType = GetSubscriptionMessageTypeFrom(context.Message);
             if (subscriptionMessageType != null)
             {
+                string returnAddress;
+                if (!context.Message.Headers.TryGetValue(Headers.SubscriberTransportAddress, out returnAddress))
+                {
+                    context.Message.Headers.TryGetValue(Headers.ReplyToAddress, out returnAddress);
```

Comment:
I don't think so. We have subscription message type header which is present only on subscribe/unsubscribe messages and the return address has to be in either `SubscriberTransportAddress` (V6+) or `ReplyToAddress` (V5-). In the production code I would check for it but here in ATs I assume that if we have a bug and we don't add one of these headers, we deserved to get a null reference exception here.